### PR TITLE
Makefile changes to add targets for building local image and gofmt verification

### PR DESCRIPTION
**Pull request by ramr**
Add makefile targets to build local images and verifying go code with gofmt.
Plus fix the gofmt issue that this found. 
@pravisankar  PTAL Thx

/cc @openshift/sig-network-edge

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,14 +30,37 @@ generate: $(GOBINDATA_BIN)
...
 	go get -u github.com/jteeuwen/go-bindata/...
 
-test:
+test:	verify
 	go test ./...
 
 test-integration:
 	go test -v -tags integration ./test/integration
 
+verify:	verify-gofmt
```
Comment (ironcladlou) [inline Makefile]: is this formatting error a subtle joke?

Comment (ramr) [reply Makefile]: hmm, something weird there - it shows up as tab in vi.  Something expanding it on github?

Comment (ironcladlou) [reply Makefile]: Are you suggesting a tab there was *intentional*? I suspect we'd have a lively argument over a `make fmt` command... 😁

**Comment by ironcladlou** [comment]
Thanks!

/lgtm

File: Makefile (hunk 1 above)
Comment (ramr) [reply Makefile]: lol

**Pull request merged**